### PR TITLE
Makes the Danger title come first, not last, in the std out

### PR DESCRIPTION
**Pull request by orta**
screen shot 2018-04-20 at 9 33 34 am

felt backwards

### CODE REVIEW

**Comment by sunshinejr** [comment]
Hmm, theoretically yes, but right now when you have a long build log it is a lot easier (at least for me) to find the summary - it's near end so you can quickly see the results.

**Comment by orta** [comment]
I agree, was thinking about this afterward,  probably worth doing a check how many lines of text are in the violations, and if it's over ~10 have the message at the bottom

**Comment by sunshinejr** [comment]
Yeah, I like it more 👍

File: source/runner/Executor.ts [hunk 1]
```diff
@@ -148,11 +148,40 @@ export class Executor {
...
+        output = `Danger: ${tick} found only warnings, ${message}`
+      } else if (messages.length > 0) {
+        output = `Danger: ${tick} passed, found only messages.`
+      } else if (!messages.length && !fails.length && !messages.length && !warnings.length) {
+        output = `Danger: ${tick} passed review, received no feedback.`
+      }
+
+      const allMessages = [...fails, ...warnings, ...messages, ...markdowns].map(m => m.message)
+      const oneMessage = allMessages.join("\n")
+      const longMessage = oneMessage.split("\n").length > 30
```
Comment (orta) [inline source/runner/Executor.ts]: My laptop full screen is 40 lines, so this is 3/4 of that

Comment (sunshinejr) [reply source/runner/Executor.ts]: cool 👍

**Pull request merged**